pkgs/tools/networking/pacproxy/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "pacproxy";
+  version = "2.0.5";
+
+  src = fetchFromGitHub {
+    owner = "williambailey";
+    repo = "pacproxy";
+    rev = "v${version}";
+    hash = "sha256-oDSptPihrDIiTCgcP4t2J3vJBNGMViyPAAmBv4ynLNU=";
+  };
+
+  vendorHash = "sha256-0Go+xwzaT1qt+cJfcPkC8ft3eB/OZCvOi2Pnn/A/rtQ=";
+
+  meta = with lib; {
```

Comment:
Would you please add the changelog to the mata?

Reply:
I didn't find a changelog, but I saw other packages was using the release page, so I went ahead with that as well.